content/en/flux/components/kustomize/options.md

```diff
@@ -38,11 +38,12 @@ please see the [bootstrap customization guide](/flux/installation/configuration/
 | `--watch-label-selector`              | string        | Watch for resources with matching labels e.g. 'sharding.fluxcd.io/shard=shard1'.                                                                                                         |
 | `--feature-gates`                     | mapStringBool | A comma separated list of key=value pairs defining the state of experimental features.                                                                                                   |
 | `--interval-jitter-percentage`        | uint8         | Percentage of jitter to apply to interval durations. A value of 10 will apply a jitter of +/-10% to the interval duration. It cannot be negative, and must be less than 100. (default 5) |
+| `--override-manager`                  | []string      | Field manager disallowed to perform changes on managed resources.                                                                                                                        |
```

Comment:
Is `[]string` how it's printed by `--help`?

Reply:
Ah no it's `stringArray`